ibis/backends/sqlite/tests/test_client.py

```diff
@@ -75,3 +76,14 @@ def test_connect(url, ext, tmp_path):
     con = ibis.connect(url(path))
     one = ibis.literal(1)
     assert con.execute(one) == 1
+
+
+def test_has_operation(con):
+    # Handled by hardcoded rewrite
+    assert con.has_operation(ops.Project)
+    # Handled by base class rewrite
+    assert con.has_operation(ops.Capitalize)
+    # Handled by compiler-specific rewrite
+    assert con.has_operation(ops.Sample)
+    # Handled by visit_* method
+    assert con.has_operation(ops.Cast)
```

Comment:
We're probably going to start rewriting `Aggregation` to `Select` so we can merge aggs with subqueries where possible, is there a test we can add that will fail if/when we do that?

Reply:
Could just add `Aggregate` to the ops checked here. Will do.